<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMeasureSettings</class>
 <widget class="QWidget" name="FormMeasureSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>407</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>374</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>178</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>178</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Motor Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="spinMotorPolesCount">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <italic>false</italic>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spinMotorKV">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editMotorName">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editPropName">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="editComment">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Propeller Name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinBladeCount">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <italic>false</italic>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Motor Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Motor KV</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Motor Poles Count</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Blade count</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_4</zorder>
     <zorder>label_2</zorder>
     <zorder>label</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
     <zorder>editPropName</zorder>
     <zorder>editComment</zorder>
     <zorder>label_6</zorder>
     <zorder>editMotorName</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>77</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>77</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QCheckBox" name="checkBox_CurrZero">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Current sensor with zero</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_EscTelem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Enable ESC Telemetry</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>77</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>RPM source</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_Sensor">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>IR sensor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_Telem">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>ESC Telemetry</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editMotorName</tabstop>
  <tabstop>spinMotorKV</tabstop>
  <tabstop>spinMotorPolesCount</tabstop>
  <tabstop>editPropName</tabstop>
  <tabstop>spinBladeCount</tabstop>
  <tabstop>editComment</tabstop>
  <tabstop>checkBox_CurrZero</tabstop>
  <tabstop>checkBox_EscTelem</tabstop>
  <tabstop>radioButton_Sensor</tabstop>
  <tabstop>radioButton_Telem</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
